react/backstop/storyBookBackstop.js

```diff
@@ -91,20 +99,32 @@ const mapComponents = (components, debug) => components.map((component) => {
 
   return makeScenario(
     `${kind}/${name}`,
-    `${urlBase}iframe.html?selectedKind=${kind}&selectedStory=${name}`
+    `${urlBase}iframe.html?selectedKind=${kind}&selectedStory=${name}`,
+    viewports
   );
 });
 
+const isAtom = (component) => {
+  const { filePath } = component;
+  return(filePath.indexOf('/atoms/') > -1)
+    && (filePath.indexOf('table') === -1)
+    && (path.basename(filePath) !== 'Image')
+    && (filePath.indexOf('icons') === -1);
```

Comment:
These exceptions could probably use comments explaining why they are needed, as they had in the previous version.